apps/node-fastify/src/utils/config.ts

```diff
@@ -40,6 +43,7 @@ export function getConfig(): configType {
     DATABASE_URL: getConfigFromEnv('DATABASE_URL'),
     ORB_WEBHOOK_SECRET: getConfigFromEnv('ORB_WEBHOOK_SECRET'),
     PORT: Number(getConfigFromEnv('PORT', '8080')),
+    VERIFY_WEBHOOK_SIGNATURE: getConfigFromEnv('VERIFY_WEBHOOK_SIGNATURE', 'true') === 'true',
```

Comment:
Looks like this flag is unused?

Reply:
Is used in `app.ts` and `orb-sync.ts`. I set the respective variable in `.env` for local testing.